### ISSUE TITLE
Any hope of it coming back?

### ISSUE THREAD

**Issue by asiekierka** [Hurricaaane/MAtmos#19]
I miss you, MAtmos!

**Comment by 008** [comment]
where are sound packs ?
1.7.10 plz

**Comment by Brottweiler** [comment]
Nope, you will have to move to the forked version here: https://github.com/dags-/MC-Commons

[Reason](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/1274487-matmos-environmental-sound-atmosphere-simulator?comment=3897)

**Comment by xJon** [comment]
@asiekierka If you really want, you can make it work with 1.7.10, all the way up to 1.10.2, and I've seen some work done on 1.12.2. But aren't you the magic mod-reviver we all want and need but don't deserve? (also thanks again for helping with Factorization).

**Comment by asiekierka** [comment]
Well, I did post that over four years ago...

**Comment by xJon** [comment]
@asiekierka No time to waste then it's already been 4 years!
